sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -162,8 +162,7 @@ def __init__(
       control_address,  # type: str
       credentials=None,  # type: Optional[grpc.ChannelCredentials]
       worker_id=None,  # type: Optional[str]
-      # Caching is disabled by default
-      state_cache_size=0,  # type: int
+      state_cache_size=100 << 20,  # type: int
```

Comment:
is this sufficient? I thought we'd need to change `_get_state_cache_size` instead.

Reply:
Yes, that is right. But first, i wanted to fix on a name. I need to remove few comments regarding the state cache as well. 

Also, do we want to put state cache pipeline option in the `experiments`?

Reply:
no, a pipeline option doesn't have to be an experiment.

Reply:
I think having a pipeline option makes sense.

Reply:
naming is tricky - what is worker? is cache per entire VM (answer, for python it is per process).. might be better to iron that out in a doc.